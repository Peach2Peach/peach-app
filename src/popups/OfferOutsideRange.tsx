import PeachText from '../components/text/Text'
import i18n from '../utils/i18n'
import { offerIdToHex } from '../utils/offer/offerIdToHex'

type Props = {
  offerId: string
}

export const OfferOutsideRange = ({ offerId }: Props) => (
  <PeachText>{i18n('notification.offer.outsideRange.text', offerIdToHex(offerId))}</PeachText>
)
