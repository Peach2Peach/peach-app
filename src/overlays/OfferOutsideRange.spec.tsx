import { createRenderer } from 'react-test-renderer/shallow'
import { OfferOutsideRange } from './OfferOutsideRange'

describe('OfferOutsideRange', () => {
  const shallowRenderer = createRenderer()
  it('should render correctly', () => {
    shallowRenderer.render(<OfferOutsideRange offerId="123" />)
    expect(shallowRenderer.getRenderOutput()).toMatchSnapshot()
  })
})
