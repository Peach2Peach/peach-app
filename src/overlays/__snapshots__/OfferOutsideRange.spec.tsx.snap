// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferOutsideRange should render correctly 1`] = `
<PeachText>
  Offer P‑7B is now outside of the trading range and will not be shown to buyers.

You can choose to cancel the offer and make a new one, or leave it as is.
</PeachText>
`;
